app/code/Magento/Sales/Test/Mftf/Test/StorefrontOrderCommentWithHTMLTagsDisplayTest.xml

```diff
@@ -0,0 +1,64 @@
+    <test name="StorefrontVerifyOrderHistoryCommentsWithHTMLTagsTest">
+        <annotations>
+            <features value="Sales"/>
+            <stories value="Storefront Customer Order History Comments"/>
+            <title value="Verify Customer Order History Comments On Storefront"/>
+            <description value="Verify that comment with HTML tag is displayed properly on the Order View Page on Storefront"/>
+            <testCaseId value="MC-42694"/>
+            <useCaseId value="MC-42531"/>
+            <severity value="MINOR"/>
+            <group value="Sales"/>
+        </annotations>
+        <before>
+            <createData entity="Simple_US_Customer" stepKey="createCustomer"/>
+            <createData entity="SimpleProduct2" stepKey="createProduct"/>
+            <createData entity="CustomerCart" stepKey="createCustomerCart">
+                <requiredEntity createDataKey="createCustomer"/>
+            </createData>
+            <createData entity="CustomerCartItem" stepKey="addCartItem">
+                <requiredEntity createDataKey="createCustomerCart"/>
+                <requiredEntity createDataKey="createProduct"/>
+            </createData>
+            <createData entity="CustomerAddressInformation" stepKey="addCustomerOrderAddress">
+                <requiredEntity createDataKey="createCustomerCart"/>
+            </createData>
+            <updateData createDataKey="createCustomerCart" entity="CustomerOrderPaymentMethod" stepKey="submitOrder">
+                <requiredEntity createDataKey="createCustomerCart"/>
+            </updateData>
+        </before>
+        <after>
+            <actionGroup ref="StorefrontCustomerLogoutActionGroup" stepKey="customerLogout"/>
+            <actionGroup ref="AdminLogoutActionGroup" stepKey="logout"/>
+            <deleteData createDataKey="createCustomer" stepKey="deleteCustomer"/>
+            <deleteData createDataKey="createProduct" stepKey="deleteProduct"/>
+        </after>
+
+        <actionGroup ref="AdminLoginActionGroup" stepKey="loginAsAdmin"/>
+
+        <actionGroup ref="AdminOpenOrderViewPageByOrderIdActionGroup" stepKey="openOrder">
+            <argument name="orderId" value="$createCustomerCart.return$"/>
+        </actionGroup>
+
+        <actionGroup ref="AdminSalesOrderCommentsActionGroup" stepKey="fillAndSaveOrderComments">
+            <argument name="comment" value="{{SalesOrderComments.commentWithHyperlink}}" />
+        </actionGroup>
+
+        <actionGroup ref="LoginToStorefrontActionGroup" stepKey="loginToStorefront">
+            <argument name="Customer" value="$$createCustomer$$"/>
+        </actionGroup>
+
+        <actionGroup ref="StorefrontClickViewOrderLinkOnMyOrdersPageActionGroup" stepKey="clickViewOrder"/>
+
+        <dontSee userInput="{{anchorTagFragment.anchorTag}}" selector="{{StorefrontCustomerOrderViewSection.orderComment}}" stepKey="dontSeeExposedHtmlCode"/>
```

Comment:
I think can move to action group

Reply:
@edenduong I checked - there are no more tests that use this assertion. Do you think we still need create a separate Action Group here?
Thanks.

Reply:
@AnnaAPak It could be groundwork for the future :)